pages/api/fetch-registry-data/[...name].js

```diff
@@ -19,13 +19,19 @@ async function fetchRegistryData(packageName) {
   // Otherwise, refetch
   try {
     const response = await fetch(registryUrl)
+    // If status is not 200, throw an error
+    if (response.status !== 200) {
```

Comment:
Ran into an issue here since the package isn't published yet. I think it makes sense here to bail and throw an error if the npm registry URL responds with anything other than a successful response.